ArduCopter/Copter.cpp

```diff
@@ -514,25 +514,7 @@ void Copter::one_hz_loop()
 // called at 50hz
 void Copter::update_GPS(void)
```

Comment:
If this is the change set, shouldn't this whole function be removed and instead replace the scheduler entry with:

`SCHED_TASK_CLASS(AP_GPS,      &Copter.gps,  update, 50, 300),`

Reply:
@WickedShell Yes, that makes sense.